packages/unplugin-typia/src/core/typia.ts

```diff
@@ -118,17 +118,28 @@ async function getProgramAndSource(
 	if (aliases != null && aliases.length > 0) {
 		host.resolveModuleNameLiterals = (moduleLiterals, containingFile, _, options) => {
 			return moduleLiterals.map((lit) => {
-				let module = ts.resolveModuleName(lit.text, containingFile, options, host, host.getModuleResolutionCache?.());
+				const module = ts.resolveModuleName(lit.text, containingFile, options, host, host.getModuleResolutionCache?.());
+
+				/* if module is resolved, return it */
+				if (module.resolvedModule != null) {
```

Comment:
I would prefer`if (module.resolvedModule)` here. Loose equality makes the code less readable and is not considered good practice but you decide :)

Reply:
Thank you!